CMakeLists.txt

```diff
@@ -100,6 +100,11 @@ elseif(MSVC)
     add_compile_options("/w34057")
     # Warn about signed/unsigned mismatch.
     add_compile_options("/w34245")
+
+    # Control flow guard
+    # TODO: Why do we have this? Nothing in this repo is security critical.
+    add_compile_options(/guard:cf)
```

Comment:
If I had to guess it’s because vulkaninfo ships with the vulkan runtime and needs to meet certain criteria that windows drivers have to pass

Reply:
Ah I didn't know that. I'll update the code to reflect that. Looks like you are correct from what I can tell.